File: app/src/main/java/com/example/android/popularmovies/model/MoviesResponse.java
package com.example.android.popularmovies.model;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class MoviesResponse {

    @SerializedName("page")
    private int page;
    @SerializedName("results")
    private List<Movie> results;
    @SerializedName("total_results")
    private int totalResults;
    @SerializedName("total_pages")
    private int totalPages;


    int getPage() {
        return page;
    }

    public List<Movie> getResults() {
        return results;
    }

    int getTotalResults() {
        return totalResults;
    }

    int getTotalPages() {
        return totalPages;
    }
}

File: app/src/main/kotlin/com/example/android/popularmovies/model/MoviesResponse.kt
package com.example.android.popularmovies.model

import com.google.gson.annotations.SerializedName

class MoviesResponse {

    @SerializedName("page")
    var page: Int = 0
    @SerializedName("results")
    var results: List<Movie>? = null
    @SerializedName("total_results")
    var totalResults: Int = 0
    @SerializedName("total_pages")
    var totalPages: Int = 0
}